src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -233,10 +233,14 @@ public boolean verifyTitle(String title) {
     return true;
   }
 
-  public void verifyURLcontains(String givenString) {
+  public void verifyURLcontains(String exptectedString) {
     String currentURL = driver.getCurrentUrl();
-    Assertion.assertStringContains(currentURL.toLowerCase(), givenString.toLowerCase());
-    PageObjectLogging.log("verifyURLcontains", "current url is the same as expetced url", true);
+    if (Assertion.assertStringContains(currentURL.toLowerCase(), exptectedString.toLowerCase())) {
+      PageObjectLogging.log("verifyURLcontains", "current url contains " + exptectedString, true);
+    } else {
+      PageObjectLogging
+          .log("verifyURLcontains", "current url doesn't contain " + exptectedString, true);
```

Comment:
to make it even more reusable there should be one method called:
```
isStringInURL
```
(or something like this) - which would return only true or false and then inside tests this method should be wrapped either in AssertTrue or AssertFalse (depending on the situation).